### ISSUE TITLE
Couldn't get handle: No such file or directory

### ISSUE THREAD

**Issue by antonst** [radical-cybertools/radical.pilot#332]
Having number of CU's equal 32 I get this:

2014:09:08 13:27:54 radical.pilot.MainProcess: [ERROR   ] DoesNotExist: file copy failed: mput  "//home/antons/SINON-WORK/repex-12-cu-cancel/RepEx/examples/namd/namd_scheme_3/alanin_base_10_0.namd" "/home1/02457/antontre/radical.pilot.sandbox/pilot-540de6b91d41c81abbc19fe0/unit-540de7021d41c81abbc19fec/alanin_base_10_0.namd" 
Uploading //home/antons/SINON-WORK/repex-12-cu-cancel/RepEx/examples/namd/namd_scheme_3/alanin_base_10_0.namd to /home1/02457/antontre/radical.pilot.sandbox/pilot-540de6b91d41c81abbc19fe0/unit-540de7021d41c81abbc19fec/alanin_base_10_0.namd
Couldn't get handle: No such file or directory
sftp> 

The thing is these files are actually present (these are first input files which must be transferred to remote machine) and if I run with smaller number of CU's everything will run normally.  

So bottom line is as of now input file transfer fails for me even for 32 CU's. I am not sure why this happens.

Thanks,
Antons
